### PR TITLE
Replace running on bogota to running on jakarta

### PR DESCRIPTION
**Pull request by vtomole**
Fixes https://github.com/SupertechLabs/qiskit-superstaq/issues/179

### CODE REVIEW

File: qiskit_superstaq/daily_integration_test.py [hunk 1]
```diff
@@ -51,13 +51,14 @@ def test_acer_non_neighbor_qubits_compile(provider: qss.SuperstaQProvider) -> No
...
     qc.append(qss.AceCR("-+"), [1, 2])
     qc.append(qss.AceCR("-+"), [2, 3])
-    out = provider.ibmq_compile(qc, target="ibmq_bogota_qpu")
+    out = provider.ibmq_compile(qc, target="ibmq_jakarta_qpu")
     assert isinstance(out, qss.compiler_output.CompilerOutput)
     assert isinstance(out.circuit, qiskit.QuantumCircuit)
     assert isinstance(out.pulse_sequence, qiskit.pulse.Schedule)
-    assert 5700 <= out.pulse_sequence.duration <= 7500  # 7424 as of 4/06/2022
+    print(out.pulse_sequence.duration)
+    assert 3600 <= out.pulse_sequence.duration <= 7500  # 3616 as of 6/30/2022
```
Comment (singular-value) [inline qiskit_superstaq/daily_integration_test.py:59]:
```suggestion
    assert 3000 <= out.pulse_sequence.duration <= 4000  # 3616 as of 6/30/2022
```

File: qiskit_superstaq/daily_integration_test.py [hunk 2]
```diff
@@ -51,13 +51,14 @@ def test_acer_non_neighbor_qubits_compile(provider: qss.SuperstaQProvider) -> No
     qc.append(qss.AceCR("-+"), [0, 1])
     qc.append(qss.AceCR("-+"), [1, 2])
     qc.append(qss.AceCR("-+"), [2, 3])
-    out = provider.ibmq_compile(qc, target="ibmq_bogota_qpu")
+    out = provider.ibmq_compile(qc, target="ibmq_jakarta_qpu")
     assert isinstance(out, qss.compiler_output.CompilerOutput)
     assert isinstance(out.circuit, qiskit.QuantumCircuit)
     assert isinstance(out.pulse_sequence, qiskit.pulse.Schedule)
-    assert 5700 <= out.pulse_sequence.duration <= 7500  # 7424 as of 4/06/2022
+    print(out.pulse_sequence.duration)
```
Comment (vtomole) [inline qiskit_superstaq/daily_integration_test.py:58]:
```suggestion
```

**Review by singular-value — APPROVED** [review]
lgtm

**Comment by vtomole** [comment]
@singular-value Pushed a commit to update the `app_ss` version. Will need a re-approval.

**Review by singular-value — APPROVED** [review]

**Pull request merged**